docs/imviz/import_data.rst

```diff
@@ -0,0 +1,34 @@
+.. _imviz-import-data:
+
+***********
+Import Data
+***********
+
+Imviz can load data in the form of a filename (string), .jpg, .png,
+an astropy object, or a numpy array if the data is 2D.
+
+Importing data through the GUI
+------------------------------
+
+The first way that users can load their data into the Imviz application is
+by clicking the :guilabel:`Import` button at the top left of the application's
+user interface. This opens a dialogue where the user can enter the path of file
+that can be parsed as a :class:`~astropy.nddata.NDData` (2D data only), :class:`~astropy.io.fits.HDUList`,
```

Comment:
It only works on 2D, yes, but it is not limited to NDData, so it is misleading to put the parenthesis here. I think this was already covered in the section above anyway.

```suggestion
that can be parsed as a :class:`~astropy.nddata.NDData`, :class:`~astropy.io.fits.HDUList`,
```